docs/readthedocs/source/doc/LLM/Quickstart/index.rst

```diff
@@ -8,4 +8,5 @@ BigDL-LLM Quickstart
 This section includes efficient guide to show you how to:
 
 * `Install BigDL-LLM on Windows with Intel GPU <./install_windows_gpu.html>`_
-* `Use Text Generation WebUI on Windows with Intel GPU <./webui_quickstart.html>`_
\ No newline at end of file
+* `Use Text Generation WebUI on Windows with Intel GPU <./webui_quickstart.html>`_
+* `Performance Benchmarking <./benchmark_quickstart.html>`_
```

Comment:
We may need a verb phrase here? For example, "Conduct Performance Benchmarking with BigDL-LLM"?